packages/web-components/src/message-bar/message-bar.styles.ts

```diff
@@ -0,0 +1,126 @@
+import type { ElementStyles } from '@microsoft/fast-element';
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground3,
+  colorNeutralForeground3,
+  colorNeutralStroke1,
+  colorPaletteDarkOrangeBackground1,
+  colorPaletteDarkOrangeBorder1,
+  colorPaletteGreenBackground1,
+  colorPaletteGreenBorder1,
+  colorPaletteRedBackground1,
+  colorPaletteRedBorder1,
+  fontFamilyBase,
+  fontSizeBase200,
+  lineHeightBase200,
+  spacingHorizontalM,
+  spacingHorizontalS,
+  spacingVerticalMNudge,
+} from '../theme/design-tokens.js';
+
+import { errorState, multiLineState, squareState, successState, warningState } from '../styles/states/index.js';
+
+/**
+ * Styles for the MessageBar component.
+ *
+ * @public
+ */
+export const styles: ElementStyles = css`
+  :host {
+    display: grid;
+    box-sizing: border-box;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    line-height: ${lineHeightBase200};
+    width: 100%;
+    background: ${colorNeutralBackground3};
+    border: 1px solid ${colorNeutralStroke1};
+    padding: 0 ${spacingHorizontalM};
+    border-radius: ${borderRadiusMedium};
+    min-height: 36px;
+    align-items: center;
+    grid-template: 'icon body actions close' / auto 1fr auto auto;
+  }
```

Comment:
Lets add CSS containment here - I totally missed this earlier in my review

Reply:
Done!